### DISCUSSION TITLE
is it possible to make the projects header sticky?

### DISCUSSION THREAD

**Discussion by MonsterITServices** [YetiForceCompany/YetiForceCRM#16789] [General] [answered]
So when scrolling is it possible to have the menu header sticky.
![image](https://user-images.githubusercontent.com/22075161/210372450-a5dda94b-55c9-42dc-b21a-37b3aadbe4c0.png)

So when scrolling through all records you can see what they mean?

**Comment by rskrzypczak** [comment, marked as answer]
Currently YetiForce does not have such functionality.

**Reply by MonsterITServices** [reply]
Okay thank you.
